packages/jsii-pacmak/lib/targets/go/types/class.ts

```diff
@@ -241,6 +248,10 @@ export class ClassMethod extends GoMethod {
   public get instanceArg(): string {
     return this.parent.name.substring(0, 1).toLowerCase();
   }
+
+  public get usesReflectionPackage() {
+    return Boolean(this.reference?.scopedImplMap(this.parent.pkg).length);
```

Comment:
Just double-not things to coerce them into a boolean context...

```suggestion
    return !!this.reference?.scopedImplMap(this.parent.pkg).length;
```